### PR TITLE
Added pipeline test and test history.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@lwander

### CODE REVIEW

File: spinnaker/spinnaker_system/google_front50_test.py [hunk 1]
```diff
@@ -158,32 +179,46 @@ def update_app(self):
...
+
+    for clause in gcs_builder.build().clauses:
+      contract.add_clause(clause)
+
+    f50_builder = st.http_observer.HttpContractBuilder(self.agent)
+    (f50_builder.new_clause_builder('History Records Changes')
+     .get_url_path('/default/applications/{app}/history'
+                   .format(app=self.TEST_APP))
+     .contains_path_value('[0]', self.app_history[0])
+     .contains_path_value('[1]', self.app_history[1]))
```
Comment (lwander) [inline spinnaker/spinnaker_system/google_front50_test.py]: The flow with the `self.app_history` is a little confusing - what's going on here? I see you insert expected values above, but I'm not sure what you're searching for.

Comment (ewiseblatt) [inline spinnaker/spinnaker_system/google_front50_test.py]: The historic entities were remembered properly.
Path '[N]' means the Nth element in a list. So this is checking the list reflects the value and ordering.

Comment (lwander) [inline spinnaker/spinnaker_system/google_front50_test.py]: Oh I see, makes sense, thanks

**Comment by lwander** [comment]
LGTM

**Pull request merged**